[docs] New API design rule disabled > disable (#34972)
mui/material-ui: docs/data/material/guides/api/api.md

### Prop naming

- **Boolean**

>   - The default value of a boolean prop should be `false`. This choice allows the shorthand notation For example an input is by default enabled: How to name the prop that controls this state? It should be called `disabled` because:

Reviewer: Just cleaning this up a bit.

```suggestion
  - The default value of a boolean prop should be `false`. This allows for better shorthand notation. Consider an example of an input that is enabled by default. How should you name the prop that controls this state? It should be called `disabled`:
```

    ```jsx
    ❌ <Input enabled={false} />
    ✅ <Input disabled />
    ```

>   - The name of the boolean, if composed of a single word, should be an adjective or a noun and not a verb. This is because the prop describes a state. For example an input prop can be controlled by a state, which wouldn't be called with a verb:

Reviewer: Grammar and a little more clarity.

```suggestion
  - If the name of the boolean is a single word, should be an adjective or a noun rather than a verb. This is because props describe a _state_ and not an _action_. For example an input prop can be controlled by a state, which wouldn't be described with a verb:
```
